.github/workflows/cypress.yml

```diff
@@ -80,3 +81,81 @@ jobs:
           CYPRESS_TEST_USER_EMAIL: "cypress@example.com"
           CYPRESS_TEST_USER_PASS: ${{ secrets.CYPRESS_TEST_USER_PASS }}
           CYPRESS_baseUrl: ${{ github.event.deployment_status.environment_url }}
+
+  configuremobile:
+    needs: Setup
+    runs-on: ubuntu-latest
+    outputs:
+      BUILD_MOBILE_APP: ${{ steps.checkdiff.outputs.BUILD_MOBILE_APP }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 5
+        name: check diff
+      - id: checkdiff
+        run: |
+          git fetch origin main
+          echo $(git diff --name-only origin/main -- | grep "/clients/mobile/react-native/" | wc -l)
+          echo "BUILD_MOBILE_APP=$(git diff --name-only origin/main -- | grep "/clients/mobile/react-native/" | wc -l)" >> $GITHUB_OUTPUT
```

Comment:
I think this is fine for now but we could just improve the `vue_or_react.sh` script to be more generic and just have it spit out "Whats the primary stack this PR is about". Then do cleaner `if` guards elsewhere in this file.

But again, not needed. Can refactor later.

Reply:
maybe we can move the artifacting to a parent yaml file and then call the various ones -- cypress/mobile deploy based on the script!